pom.xml

```diff
@@ -28,6 +26,13 @@
     <!-- JVM params for test execution -->
     <argLine>-Duser.language=en</argLine>
 
+    <!-- Spring Boot / Cloud / Stream deps -->
```

Comment:
@erdemedeiros we need to move this somewhere.. let's start thinking where.. maybe activiti-services??